auth0/resource_auth0_connection.go

```diff
@@ -429,6 +429,78 @@ var connectionSchema = map[string]*schema.Schema{
 					Optional:    true,
 					Description: "",
 				},
+				"debug": {
+					Type:        schema.TypeBool,
+					Optional:    true,
+					Description: "When enabled, additional debug information will be generated.",
+				},
+				"signing_cert": {
+					Type:        schema.TypeString,
+					Optional:    true,
+					Description: "X.509 signing certificate (encoded in PEM or CER) you retrieved from the IdP, Base64-encoded",
+				},
+				"protocol_binding": {
+					Type:        schema.TypeString,
+					Optional:    true,
+					Description: "The SAML Response Binding: how the SAML token is received by Auth0 from IdP",
+					ValidateFunc: validation.StringInSlice([]string{
+						"urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect",
+						"urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST",
+					}, true),
+				},
+				"idpinititated": {
```

Comment:
We could make this a little more readable:
```suggestion
				"idp_inititated": {
```